regular-expression.dd

```diff
@@ -5,128 +5,133 @@ $(D_S Regular Expressions,
     $(SMALL by Dmitry Olshansky, the author of std.regex )
     )
     $(H3 Introduction)
-    $(P String processing is a kind of daily routine that most applications do in a one way or another.
-    It should come as no wonder that many programming languages have standard libraries equipped
+    $(P String processing is a daily routine that most applications have to deal with in a one way or another.
+    It should come as no surprise that many programming languages have standard libraries equipped
     with a variety of specialized functions for common string manipulation needs.
     The D programming language standard library among others offers a nice assortment in $(STD string),
      as well as generic functions  from $(STD algorithm) that work with strings.
     Still no amount of fixed functionality could cover all needs, as naturally flexible text data
     needs flexible solutions.
     )
-    $(P Here is where $(LUCKY regular expressions) – often just called $(I regexes) – come in handy.
-     Regular expressions are a simple yet comprehensive language for defining string patterns. 
-     Combined with a mechanism for string substitution, regular expressions are a powerful tool for text processing.
-     In fact, they are considered so useful that a number of languages provide built-in support
-     for them. However, built-in support does $(I not) necessarily imply $(I faster) processing 
-     or more features, it is rather a way of providing a
-     convenient and user-friendly syntax for typical operations and usage patterns.
+    $(P This is where $(LUCKY regular expressions) come in handy, often succinctly called as regexes.
+     Regexes are simple yet powerful language for defining patterns for sets of strings.
+     Combined with pattern matching, data extraction and substitution, they form a Swiss Army knife of text processing.
+     They are considered so important that a number of programming languages provides built-in support
+     for regular expressions.  Being built-in  however does $(B not) necessary
+     implies $(B faster) processing or having more features. It's just a matter of providing
+     $(B convenient and friendly syntax) for typical operations, and integrating it well
```

Comment:
In contrast to `$(B faster) processing` plus we have raw literals which are just fine.